app/models/patient.rb

```diff
@@ -32,10 +32,16 @@ class Patient
   mongoid_userstamp user_model: 'User'
 
   # Methods
+
+
   def self.search(name_or_phone_str) # TODO: optimize
-    name_matches = Patient.where name: name_or_phone_str
-    primary_matches = Patient.where primary_phone: name_or_phone_str
-    secondary_matches = Patient.where secondary_phone: name_or_phone_str
-    (name_matches | primary_matches | secondary_matches)
+    clean_phone = name_or_phone_str.gsub(/\D/, '')
+    phone = clean_phone[0..2] + '-' + clean_phone[3..5] + '-' + clean_phone[6..10]
```

Comment:
@KatSDC What do you think about renaming `phone` to `formatted_phone`? My inclination is to err on the side of being as clear as possible, and just calling it `phone` is a little vague. Thoughts?